src/MicrosoftTeams.schema.json

```diff
@@ -247,38 +248,18 @@
                                         "properties": {
                                             "title": {
                                                 "type": "string",
-                                                "description": "The one word command name",
-                                                "maxLength": 16
-                                            },
-                                            "usageHint": {
-                                                "type": "string",
-                                                "description": "A short usage template for the command",
-                                                "maxLength": 32
-                                            },
-                                            "shortDescription": {
-                                                "type": "string",
-                                                "description": "A short description of the command",
+                                                "description": "The bot command name",
                                                 "maxLength": 32
                                             },
-                                            "value": {
-                                                "type": "string",
-                                                "description": "Payload to use for the bot action type. E.g, a URL for an OpenURL type."
-                                            },
-                                            "type": {
-                                                "type": "string",
-                                                "enum" : ["imBack", "invoke", "openURL", "prompt", "gotoTab"],
-                                                "description": "The type of bot interaction the command defines.",
-                                                "maxLength": 16
-                                            },
-                                            "image": {
+                                            "description": {
                                                 "type": "string",
-                                                "description": "An optional URL to an image to use for the command entry"
+                                                "description": "A simple text description or an example of the command syntax and its arguments.",
+                                                "maxLength": 128
```

Comment:
Same comment as above applies here? Where will this string be displayed in our UI and will it look good even at the max length?